iot/dotnet-iot-and-nanoframework/README.md

```diff
@@ -0,0 +1,67 @@
+
+| .NET Core IoT                | .NET nanoFramework |
+| :----------------------------------------------------------: | :----------------------------------------------------------: |
+| ![Raspberry Pi with .NET Core IoT](images/RP-BMP280_bb.png) | ![M5Stack with .NET nanoFramework](images/ESP32-BMP280_bb.png) |
+| [Raspberry Pi with .Net Core IoT sample project](dotnet-iot) | [M5Stack with .NET nanoFramework sample project](nanoFramework) |
+
+## Certificate
+
+Device certificates are used in these samples. Certificates provide handshake between your device and Azure IoT Hub, and samples explain how to use certificates on both .NET Core IoT and .NET nanoFramework platforms.
+
+If you don't have any certificate, you can learn how to create certificates following [Create test certificate using OpenSSL and Azure IoT Hub](create-certificate.md) documentation.
+
+## Quick Tips & Differences
+
+| .NET Core IoT                                                | .NET nanoFramework                                           |
+| ------------------------------------------------------------ | ------------------------------------------------------------ |
+| Works on powerful boards with memory and CPU                 | Works on low-level microcontroller devices with low memory   |
+| Due to device specs it can benefit from all .NET core libraries | Uses specific .NET libraries for nanoFramework due to performance on low-level devices |
+| You can add your packages via NuGet manager                  | You can add your packages via NuGet manager                  |
+| You only need to add `Iot.Device.Bindings` and `System.Device.Gpio` packages for new sensors | You need to add nanoFramework specific packages separately for each sensor |
+| You need to setup your deployment and remote debugging manually | Built-in deployment and rich remote debugging experience as Default |
+| Deploy the code via SSH connection                           | Deploy the code directly via wired connection                |
+| It can use WiFi of Raspberry Pi OS                           | You need to connect your device to WiFi through your code                   |
+| You can use `pfx` certificates in your code                  | You can use `PEM`, `crt` and `der` certificates in your code |
+
+## Code Comparison
+
+With this comparison we use exactly the same scenario for .NET Core IoT and .NET nanoFramework platforms. We're controlling BMP280 temperature & pressure sensor and LED via GPIO pins, connecting to wifi and accessing Azure IoT Hub via client specific Azure SDKs. We're using certificates to identify devices on Azure IoT Hub and grant access based on certificates.
+
+Code may change due to access some platform specific resources and SDKs, but majority of the code lines are identically same in these samples. Let's compare what are the differences on these platforms.
```

Comment:
You may add the reason as well most code is the same: API between both platforms are aligned, SDK API are aligned too and you can find Iot.Device bindings as well with aligned API. And despite the fact that .NET nanoFramework do not support (yet) generics, away and async, and is a subset of the full framework, the most important API are present and everything is aligned to make it as easy as possible to reuse a maximum of code.

Reply:
Thanks, this is added as well